entrypoint.sh

```diff
@@ -4,8 +4,11 @@ if [ "$(ls -A /home/firefox/.fonts/)" ]; then
   fc-cache -f -v
 fi
 
+echo "user_pref(\"marionette.port\", $MARIONETTE_PORT);"
+echo "user_pref(\"marionette.port\", $MARIONETTE_PORT);" >> /home/firefox/profile/prefs.js
+
 ip=$(hostname --ip-address)
 
-socat tcp-listen:2828,bind="$ip",fork tcp:127.0.0.1:2828 &
+socat tcp-listen:$MARIONETTE_PORT,bind="$ip",fork tcp:127.0.0.1:$MARIONETTE_PORT &
 
-/usr/bin/firefox -headless -marionette -safe-mode -profile /home/firefox/profile/
+/usr/bin/firefox -headless -marionette -profile /home/firefox/profile/
```

Comment:
why there should be no `-safe-mode` here anymore?

Reply:
Re-Added :)I haven't seen it changes anything too much